conftest.py

```diff
@@ -30,6 +31,13 @@ def set_seed(seed: int = 0):
     yield
 
 
+@contextmanager
+def reset_dynamo():
+    dynamo.config.cache_size_limit = 512
```

Comment:
if it does more than reset, it should be called something else, like `setup_dynamo`

Reply:
right, done